data/presets/playground/map.json

```diff
@@ -0,0 +1,20 @@
+{
+    "icon": "maki-playground",
```

Comment:
To better clarify that a playground map is a surface rather than something upright, how about using the same compass icon as the Compass Rose preset? A playground map isn’t a compass rose, but it’s very similar.

https://github.com/openstreetmap/id-tagging-schema/blob/adf729045777be2ef41bb09c6318ad17be431ffd/data/presets/man_made/compass_rose.json#L2